projects/burnr/src/components/SendFundsForm.tsx

```diff
@@ -152,7 +152,7 @@ const SendFundsForm: FunctionComponent = () => {
       const fee = await api.tx.balances
         .transfer(address, new BN(amount))
         .paymentInfo(sender)
-      setFee(fee.partialFee)
+      setFee(fee.partialFee as unknown as Balance)
```

Comment:
Are you sure there's no other solution? I would find it weird that suddenly PolkadotJS breaks their typing

Reply:
I haven't figure out any other way. `Balance` is imported for now from `@polkadot/types/interfaces`, which was working fine. I tried also importing from `@polkadot/types/interfaces/runtime` which is the one that `partialFee` corresponds to [in the API](https://github.com/polkadot-js/api/blob/master/packages/types/src/interfaces/payment/types.ts#L5) but the error keeps appearing.

Reply:
> I haven't figure out any other way

Removing the `as unkwown as Balance` will suffice. I.e this:

```ts
setFee(fee.partialFee)
```

is fine, b/c `fee`'s type is `RuntimeDispatchInfo` and the `partialFee` property's type is a `Balance` already.

Reply:
```
setFee(fee.partialFee)
```
Code was like that - but build is breaking with that, since I updated the polkadotApi

Reply:
:thinking: I checkout out your branch, then I ran `git clean -dfx && yarn` (from the root folder), then I changed that line to `setFee(feet.partialFee)` and I then ran `yarn build`... Everything works just fine :shrug:. Could you please make that change and let's see if the CI is ok with it?

Reply:
Absolutely. Thank you for the insight...